Misc/NEWS.d/next/Library/2024-09-26-14-43-32.gh-issue-102536.xLiIvi.rst

```diff
@@ -0,0 +1 @@
+Add ``_architecture`` field to :data:`sys.implementation`
```

Comment:
```suggestion
Add ``_architecture`` field to :data:`sys.implementation`.
```

We should explain what this new field holds. Is it just the CPU architecture? what is its Python type?

Reply:
Oops, picnixz, you always point out the symbols I'm missing, and that makes me feel embarrassed, haha.